sdk/keyvault/azsecrets/utils_test.go

```diff
@@ -37,7 +37,9 @@ var (
 )
 
 func TestMain(m *testing.M) {
-	vaultURL = strings.TrimSuffix(os.Getenv("AZURE_KEYVAULT_URL"), "/")
+	if recording.GetRecordMode() != recording.PlaybackMode {
```

Comment:
It's not important but I wouldn't bother changing this, because we've reached the end of feature development for this module